### PR TITLE
use absolute path

### PR DESCRIPTION
**Pull request by PumpUp**
in nodejs there is a path problem. This is a simple way to correct it.

### CODE REVIEW

File: src/drivers/phantomjs/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const
 	phantomjs = require('phantomjs-prebuilt');
 
 exports.run = function(args, callback) {
-	args.unshift.apply(args, ['driver.js', '--web-security=false', '--load-images=false', '--ignore-ssl-errors=yes', '--ssl-protocol=any']);
+	args.unshift.apply(args, [__dirname+"/"+'driver.js', '--web-security=false', '--load-images=false', '--ignore-ssl-errors=yes', '--ssl-protocol=any']);
```
Comment (hbakhtiyor) [inline src/drivers/phantomjs/index.js]: better to use `path.join(__dirname, 'driver.js')`

Comment (craiglondon) [inline src/drivers/phantomjs/index.js]: I there there is another commit pending that is trying to solve this issue as well?

Comment (hbakhtiyor) [inline src/drivers/phantomjs/index.js]: @craiglondon not sure i understand

Comment (hbakhtiyor) [inline src/drivers/phantomjs/index.js]: yeah, exactly https://github.com/AliasIO/Wappalyzer/pull/1462/

Comment (PumpUp) [inline src/drivers/phantomjs/index.js]: #1462 ok, i close this request

**Pull request closed without merging**